config/socket/game.js

```diff
@@ -32,7 +32,7 @@ function Game(gameID, io) {
   this.state = "awaiting players";
   this.round = 0;
   this.questions = null;
-  this.answers = null;
+  // this.answers = null;
```

Comment:
Oh, I see but we can't just comment this out @saheedt. Anyway I'm on a fix and I'll push it up right away, so no big deal we're good to go. 😄

Reply:
commenting out `this.answer` will break some part of the game in the front end. I've done a proper fix by pointing the method to properly push into `this.answer`. Kindly review my latest commit, it contains the fix

Reply:
oh that was my error, just checked my pr and saw i accidentally did that

Reply:
so i just uncommented `this.answer` and it passed locally, think thats a more rounded fix. should i push that?

Reply:
Haha anyway, we're good now. I added the article schema and we have 9 passing tests right now instead of the usual 7. Thank you for the good work so far bro 👍😄 @saheedt

Reply:
sweet! Thanks for your patience and guidance. 👍🏽

Reply:
Yeah @saheedt, so no need to make any further pushes, just review and let me know so I can merge. 😄

Reply:
yeah it all looks good now